### ISSUE TITLE
ID 259074: [Keyboard Navigation - Simon – Edit Panel]: Keyboard focus disappears while activating the "Remove" button present under the "Tags” section.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#72812]
### **User Experience:** 
Users with limited mobility or visual impaired who depend on keyboard will get impacted as focus will be lost whenever they perform any action on control which will block their further actions on that control and the user would need to navigate again.
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:**
1. Open URL: [play.grafana.org](https://play.grafana.org/) Sign in with valid credentials. "Home" page appears. 
2. Navigate the "Feature Showcase” pane heading and select it. 
3. Navigate the expanded Dropdown and select the "Edit" menu item. 
4. Navigate all the controls on "Grafana Play Home/Edit Panel" page. 
5. Navigate to “Tags” section on the right side and add some tag. Remove added tag. 
6. Observe the keyboard focus and verify the issue.

**Actual Result:**
Issue: Keyboard Focus disappears while activating the "remove" button of added tag such as "title" and further pressing tab key, the focus visible on first added tag “Demo”. 
Screen reader behavior: Screen reader announces only the page information as "Grafana Play Home- Grafana" on removing any tag/activating the "remove" button.

**Expected Result:**
Keyboard Focus should be logical and should not disappear while activating the "Remove" button of added tag such as "title" present under the "Tags" section. The focus should automatically move either on “last added tag” or the "New tag (enter key to add)" text box. There should be no need to press the tab key.  

_Note: 
The same Issue exists while adding any tag and deleting any tags throughout the application. 
The same issue while invoking "clear Filter" button present under " Alert rules" tab. 
Path: Alerting>Alert Rules>Search by data source>Clear data. 
Same issue exists with explore->inspector->query->refresh 
Same issue exists with Apply time range ->Search quick ranges_

### **Severity**:
`SEV3`

### **Area:**
`TagsInput Component (Saga UI Design System)`

```[tasklist]
### Tasks
- [ ] https://github.com/grafana/grafana/issues/74448
- [ ] https://github.com/grafana/grafana/issues/74446
- [ ] https://github.com/grafana/grafana/issues/74441
- [ ] Time Picker: Search quick ranges
- [ ] https://github.com/grafana/grafana/issues/83600
```

**Comment by grafanabot** [comment]
@eledobleefe please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one label that best describes the product or feature area. Please also add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one label that best describes the product or feature that you are discussing (e.g. `area/alerting`, `datasource/loki`, `type/docs`, `type/bug`, etc). [Our issue triage](https://github.com/grafana/grafana/blob/main/ISSUE_TRIAGE.md#3-categorizing-an-issue) doc also provides additional guidance on labeling. :rocket:

 Thank you! :heart:
